core/src/main/java/org/arquillian/cube/impl/client/container/remote/CubeAuxiliaryArchiveAppender.java

```diff
@@ -12,10 +15,17 @@
 
     @Override
     public Archive<?> createAuxiliaryArchive() {
-        return ShrinkWrap.create(JavaArchive.class, "arquillian-cube.jar")
+        JavaArchive javaArchive = ShrinkWrap.create(JavaArchive.class, "arquillian-cube.jar")
                 .addPackage(CubeController.class.getPackage())
                 .addPackages(true, CubeRemoteExtension.class.getPackage())
                 .addClass(CubeControllerProvider.class)
                 .addAsServiceProvider(RemoteLoadableExtension.class, CubeRemoteExtension.class);
+
+        if (LoadableExtension.Validate.classExists("org.arquillian.cube.requirement.ArquillianConditionalRunner") &&
+                LoadableExtension.Validate.classExists("org.arquillian.cube.spi.requirement.Requires")) {
```

Comment:
yes but I don't want to assume things. so it's always good to check it even if it's there always